src/models/userDao.js

```diff
@@ -30,15 +31,46 @@ const createUser = async (
       gender,
       password,
       profile_image,
-      age
-    ) VALUES (?, ?, ?, ?, ?, ?)
+      age,
+      admin
+    ) VALUES (?, ?, ?, ?, ?, ?,?)
     `,
-    [userId, nickname, gender, hashedPw, profileImg, age]
+    [userId, nickname, gender, hashedPw, profileImg, age, admin]
   );
 
   return result;
 };
 
+const signIn = async ( userId ) => {
+  try {
+      const [user] = await database.query(
+          `SELECT
+              *
+          FROM users
+          WHERE user_id = ? `,
+          [userId]
+      );
+      return user;
+  } catch (err) {
+      const error = new Error(`INVALID_DATA_INPUT`);
+      error.statusCode = 500;
+      throw error;
+  }
+}
+
+const getAdmin = async (userId) => {
+  const result = await database.query(
+    `SELECT 
+            *
+        FROM stores, users
+        WHERE users.id = stores.admin_user_id AND users.id=?
+        `,
+    [userId]
+   
+  );
+  return result;
+}
```

Comment:
피드백 1. 결과적으로는 user(admin) 정보를 반환해주는 것이 목적이기 때문에, users table을 기준으로 쿼리문을 작성하는 것이 가독성에 좋습니다.

피드백 2. 위에서 작성하신 SQL문은 Cross Join한 결과에 WHERE문을 사용하고 있습니다. 이는 오래된 방식이며, JOIN문을 사용해서 원하는 결과를 만들어줄 수 있도록 하면 좋겠습니다.

피드백 3. `*` 와일드카드를 사용하기 보다는 명확하게 u.id, u.name 등 가져오고자 하는 컬럼명을 명시해주세요.
 
```suggestion
const getAdminUser = async (userId) => {
  const result = await database.query(
    `SELECT 
          u.id,
          u.name,
          ...
      FROM users u
      JOIN stores s ON s.admin_user_id = u.id
      WHERE u.id = ?
        `,
    [userId]
   
  );
  return result;
}
```